### PR TITLE
prometheusadapter: Bump chart and fix metric labels

### PR DESCRIPTION
**Pull request by gracedo**
https://jira.mesosphere.com/browse/DCOS-62708 Metric label names changed in k8s 1.16+, accounting for those here. I bumped the revision here but I'm not actually sure what the rules around that are, so let me know if I did that correctly!

Currently blocked on upstream chart https://github.com/helm/charts/pull/19961. Once that is merged, this PR is unblocked.

## Testing
TODO

### CODE REVIEW

File: addons/prometheusadapter/0.5.x/prometheusadapter-2.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+kind: Addon
+metadata:
+  name: prometheusadapter
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: prometheusadapter
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.5.0-2"
+    appversion.kubeaddons.mesosphere.io/prometheusadapter: "0.5.0"
+    values.chart.helm.kubeaddons.mesosphere.io/prometheusadapter: "https://raw.githubusercontent.com/helm/charts/db62a6c595bbd9904014083edd0faa14de4096b2/stable/prometheus-adapter/values.yaml"
```
Comment (gracedo) [inline addons/prometheusadapter/0.5.x/prometheusadapter-2.yaml]: TODO: update this once https://github.com/helm/charts/pull/19961 is merged

Comment (jieyu) [reply addons/prometheusadapter/0.5.x/prometheusadapter-2.yaml]: Let's just update prometheusadapter-1.yaml given 1.3 is not yet GA'ed

File: addons/prometheusadapter/0.5.x/prometheusadapter-1.yaml [hunk 2]
```diff
@@ -53,7 +53,7 @@ spec:
                  resource: namespace
                 pod_name:
```
Comment (samvantran) [inline addons/prometheusadapter/0.5.x/prometheusadapter-1.yaml]: should this not also be renamed to `pod`? Basing off [these changes](https://github.com/helm/charts/pull/19961/files#diff-c22b7d7670f281dfec8cc6b348229a1eR77)

File: addons/prometheusadapter/0.5.x/prometheusadapter-1.yaml [hunk 3]
```diff
@@ -65,5 +65,5 @@ spec:
...
```
Comment (samvantran) [inline addons/prometheusadapter/0.5.x/prometheusadapter-1.yaml]: same here

File: addons/prometheusadapter/0.5.x/prometheusadapter-1.yaml (hunk 2 above)
Comment (jieyu) [reply addons/prometheusadapter/0.5.x/prometheusadapter-1.yaml]: Good catch!

Comment (jieyu) [reply addons/prometheusadapter/0.5.x/prometheusadapter-1.yaml]: I pushed a fix to this branch

**Comment by jieyu** [comment]
I launched a cluster with konvoy master using this branch of addons.

```
➜  konvoy git:(master) k get pods.metrics.k8s.io --all-namespaces | head
NAMESPACE               NAME                                                                AGE
kubeaddons              external-dns-kubeaddons-64dc9cb45f-d2xhs                            0s
kube-system             calico-kube-controllers-5f59479c87-gbzm7                            0s
kubeaddons              dex-kubeaddons-548764787d-xrsq5                                     0s
kube-system             kube-controller-manager-ip-10-0-193-61.us-west-2.compute.internal   0s
kommander               kcl-tfcb-9c88c7bc9-xcsbm                                            0s
kube-system             kube-proxy-zd64d                                                    0s
kubeaddons              opsportal-kubeaddons-kommander-7c87c954bc-m6vcq                     0s
velero                  velero-kubeaddons-5f9dc9c54b-l9s5w                                  0s
cert-manager            cert-manager-kubeaddons-7d7f98fbc6-jsvhg                            0s
```

**Pull request merged**